tests/test_setup_config.py

```
@@ -1414,6 +1430,73 @@ def test_non_windows_platform_no_modules(mocker, reload_setup_config_module):
+    """Test _setup_github when token exists and user keeps it."""
+    from fetchtastic.setup_config import _setup_github
+
+    config = {"GITHUB_TOKEN": "fake_existing_token_12345678901234567890"}
+
+    # Mock input to keep existing token
+    monkeypatch.setattr("builtins.input", lambda _: "n")
+
+    result = _setup_github(config)
+
+    assert result is config
+    assert result["GITHUB_TOKEN"] == "fake_existing_token_12345678901234567890"
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "GITHUB_TOKEN"

(S105)